kotlin/workflow-core/src/main/java/com/squareup/workflow/WorkflowContext.kt

```diff
@@ -28,7 +28,7 @@ import kotlinx.coroutines.experimental.suspendCancellableCoroutine
  *
  * ## Handling Events
  *
- * See [makeSink], [makeUnitSink].
+ * See [onEvent], [makeUnitSink].
```

Comment:
You killed `makeUnitSink`, right? Still a lot of references to it.

Reply:
You mean we have comments? How did those get in there!